### PR TITLE
Updated .gitignore.

### PR DESCRIPTION
**Pull request by gyliu513**
/sig multicluster

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,7 @@
 /bazel-*
 examples/samplecontainer/contents/clusterregistry
 examples/samplecontainer/contents/slackcontroller
+
+vendor
```
Comment (perotinus) [inline .gitignore]: Maybe `vendor/*`?

**Comment by perotinus** [comment]
@gyliu513 This looks good overall, but I have one small comment.

File: .gitignore (hunk 1 above)
Comment (gyliu513) [reply .gitignore]: Done

**Pull request merged**